build.sbt

```diff
@@ -75,7 +75,11 @@ lazy val `scalajs-env-jsdom-nodejs`: Project = project.in(file("jsdom-nodejs-env
       "org.scala-js" %% "scalajs-env-nodejs" % scalaJSVersion,
 
       "com.novocode" % "junit-interface" % "0.11" % "test",
-      "org.scala-js" %% "scalajs-js-envs-test-kit" % scalaJSVersion % "test"
+      "org.scala-js" %% "scalajs-js-envs-test-kit" % scalaJSVersion % "test",
+
+      // See JSDOMNodeJSEnvTest.reactUnhandledExceptionHack
+      "org.webjars.npm" % "react" % "16.13.1" % "test" intransitive(),
+      "org.webjars.npm" % "react-dom" % "16.13.1" % "test" intransitive(),
```

Comment:
Why `intransitive()`? This is just in the test configuration anyways, no?

Reply:
This means that we don't need the dependencies *of* `react` and `react-dom`. We don't need any of the dependencies that they declare.

I would just have let them resolve anyway, but one of them simply refuses to resolve (https://github.com/webjars/webjars/issues/1789), so `intransitive()` fixes that.

Reply:
Ah, right. Maybe add this as a comment?